packages/common/src/models/Track.ts

```diff
@@ -219,6 +219,7 @@ export type TrackMetadata = {
   tags: Nullable<string>
   title: string
   track_segments: TrackSegment[]
+  artwork: CoverArtSizes & { mirrors?: string[] | undefined }
```

Comment:
If it's optional, why not just add it to `CoverArtSizes` directly?

Reply:
b/c CoverArtSizes is used for playlists too! coming in future PR!